### PR TITLE
OperatorProxy

### PR DESCRIPTION
**Pull request by alexroan**
- `OperatorProxy` contract deployed by `Operator`
- `forward` function moved from `Operator` to `OperatorProxy`
- Naming change `Oracle...` -> `Operator...`

### CODE REVIEW

File: evm-contracts/src/v0.7/dev/OperatorProxy.sol [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import "./LinkTokenReceiver.sol";
+
+contract OperatorProxy is Owned {
+
+  constructor() Owned(msg.sender) {}
+
+  function forward(address _to, bytes calldata _data) public
+  {
+    require(OracleInterface(owner).isAuthorizedSender(msg.sender), "Not an authorized node");
+    require(_to != LinkTokenReceiver(owner).getChainlinkToken(), "Cannot send to Link token");
```
Comment (se3000) [inline evm-contracts/src/v0.7/dev/OperatorProxy.sol:14]: Either in this PR or the follow up, we should set the LINK token address as an immutable field on here. That will save ~1500 gas on every call.

File: evm-contracts/src/v0.7/dev/OperatorProxy.sol [hunk 2]
```diff
@@ -0,0 +1,18 @@
+pragma solidity 0.7.0;
+
+import "./Owned.sol";
+import "../interfaces/OracleInterface.sol";
```
Comment (se3000) [inline evm-contracts/src/v0.7/dev/OperatorProxy.sol:4]: Let's update this to OperatorInterface to reflect our current naming patterns.

File: evm-contracts/src/v0.7/dev/Operator.sol [hunk 3]
```diff
@@ -70,6 +71,7 @@ contract Operator is
     Owned(owner)
   {
     linkToken = LinkTokenInterface(link); // external but already deployed and unalterable
+    new OperatorProxy(link);
```
Comment (alexroan) [inline evm-contracts/src/v0.7/dev/Operator.sol:74]: Is there any need for the `Operator` to store a reference to `OperatorProxy` @se3000 ?

**Comment by alexroan** [comment]
Closing due to a change of scope

**Pull request closed without merging**